totalRP3/core/impl/communication_protocol_broadcast.lua

```diff
@@ -320,20 +361,27 @@ Comm.broadcast.init = function()
 			-- We'll send out the event nice and early to say we're setting up.
 			TRP3_API.events.fireEvent(TRP3_API.events.BROADCAST_CHANNEL_CONNECTING);
 
+			-- Allow to join the broadcast channel even if the channel list is not ready
+			-- after 10 seconds.
+			local canJoinWhenNotReady = false;
+			C_Timer.After(10, function() canJoinWhenNotReady = true end);
+
 			local firstTime = true;
 			ticker = C_Timer.NewTicker(1, function(_)
 				if firstTime then firstTime = false; return; end
 				if GetChannelName(string.lower(config_BroadcastChannel())) == 0 then
-					Log.log("Step 1: Try to connect to broadcast channel: " .. config_BroadcastChannel());
-					JoinChannelByName(string.lower(config_BroadcastChannel()));
+					if canJoinWhenNotReady or isChannelListReady() then
```

Comment:
Fix up the renamed variable.

```suggestion
					if forceJoinChannel or isChannelListReady() then
```